flink-cdc-composer/src/main/java/org/apache/flink/cdc/composer/definition/PipelineDef.java

```diff
@@ -150,19 +148,6 @@ public static void validatePipelineDefinition(Configuration configuration)
                         PipelineOptions.PIPELINE_SCHEMA_OPERATOR_UID,
                         PipelineOptions.PIPELINE_SCHEMA_OPERATOR_RPC_TIMEOUT);
 
-        Set<String> optionKeys =
-                options.stream().map(ConfigOption::key).collect(Collectors.toSet());
-
-        configuration
-                .getKeys()
-                .forEach(
-                        key -> {
-                            if (!optionKeys.contains(key)) {
-                                throw new ValidationException(
-                                        String.format("Unknown configuration key `%s`", key));
-                            }
-                        });
-
         options.forEach(
                 option -> {
                     if (!configuration.getOptional(option).isPresent()
```

Comment:
To be honest, I also don't understand why unspecified configuration without default value is error?

Reply:
Since required / optional configuration in pipeline is vague (and undocumented), I've removed all pipeline-level validations.